python/TestHarness/testers/SchemaDiff.py

```diff
@@ -0,0 +1,159 @@
+#* This file is part of the MOOSE framework
+#* https://www.mooseframework.org
+#*
+#* All rights reserved, see COPYRIGHT for full restrictions
+#* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+#*
+#* Licensed under LGPL 2.1, please see LICENSE for details
+#* https://www.gnu.org/licenses/lgpl-2.1.html
+
+from RunApp import RunApp
+from TestHarness import util
+import json, os
+
+class SchemaDiff(RunApp):
+    @staticmethod
+    def validParams():
+        params = RunApp.validParams()
+        params.addRequiredParam('schemadiff',   [], "A list of XML or JSON files to compare.")
+        params.addParam('gold_dir',      'gold', "The directory where the \"golden standard\" files reside relative to the TEST_DIR: (default: ./gold/). This only needs to be set if the gold file is in a non-standard location.")
```

Comment:
It's "gold standard". See https://pubmed.ncbi.nlm.nih.gov/16402524/

Reply:
This is actually copied straight from the old XMLDiffer, I didn't write it. I can change it though.

Reply:
yes, please, otherwise the git blame will still indicate that you are responsible for this :-)